returnn/tf/layers/rec.py

```diff
@@ -295,6 +295,7 @@ def transform_config_dict(cls, d, network, get_layer):
     :param ((str) -> LayerBase) get_layer: function to get or construct another layer
     """
     if isinstance(d.get("unit"), dict):
+      d.setdefault("from", [])
```

Comment:
Let this be handled eventually by #519 but not here. This is inconsistent. Also a RecLayer with subnet can have an input (it would unpack it inside the subnet). This is a very valid case which we use often.

Reply:
So instead, just set explicitly for now in the test and in your config.